src/content/docs/browser/new-relic-browser/browser-pro-features/intro-to-browser-pro-features.mdx

```diff
@@ -0,0 +1,12 @@
+---
+title: Browser Pro agent features
+metaDescription: 'The features associated with the New Relic Browser Pro agent'
+---
+
+There are three [types of New Relic browser agent](/docs/browser/browser-monitoring/installation/install-browser-monitoring-agent#agent-types): Lite, Pro, and Pro+SPA. When enabling New Relic browser monitoring, by default the Pro+SPA browser agent is installed but some customers will have another agent type because: 
+
+* Some older customers started out with another version, or
```

Comment:
I don't think you literally mean "older customers." :-) I think you mean "older customer accounts"